main.go

```
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+type ServerResponce struct {
+	Host       string `json:"host"`
+	UserAgent  string `json:"user_agent"`
+	RequestURI string `json:"request_uri"`
+	Headers    Header `json:"headers"`
+}
+
+type Header struct {
+	Accept    string `json:"Accept"`
+	UserAgent string `json:"User-Agent"`
+}
+
+func handleRoot(w http.ResponseWriter, r *http.Request) {
+	var resp ServerResponce
+
+	resp.Host = r.Host
+	resp.UserAgent = r.UserAgent()
+	resp.RequestURI = r.RequestURI
+	resp.Headers.Accept = r.Header.Get("Accept")
+	resp.Headers.UserAgent = r.Header.Get("User-Agent")
+
+	w.Header().Set("Content-Type", "application/json; charset-UTF-8")
+	w.WriteHeader(http.StatusOK)
+	res, _ := json.Marshal(resp)

```

Why do you ignore an error returned by `Marshal` function ?